File: CodeForces/Special/cf-Goodbye2019/A.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int T;scanf("%d",&T);
    int n,a,b;

    while(T--) {
        scanf("%d%d%d",&n,&a,&b);

        int p1[a+1];
        int p2[b+1];
        
        int x=1,y=1;
        for(int i=1;i<=a;i++) {
            scanf("%d",&p1[i]);
            x=max(x,p1[i]);
        }
        for(int i=1;i<=b;i++) {
            scanf("%d",&p2[i]);
            y=max(y,p2[i]);
        }

        if(x<y) printf("NO\n");
        else printf("YES\n");
    }

    return 0;
}

File: Codeforces/Special/cf-Goodbye2019/a.py
import sys; import math; 
CASE=1; rla=''



def solve() : 
    n, o, p=map(int, input().split())
    la=[int(i) for i in input().split()]; lb=[int(i) for i in input().split()]
    print("YES" if max(la)>max(lb) else "NO")

while True : 
    rla=sys.stdin.readline()
    cases=1
    if not rla: break
    if (rla=='\n')|(rla=='') : continue
    if CASE==1 : cases=int(rla)
    for cas in range(cases) : 
        
        solve()
    

